civihr_employee_portal/views/views_export/views_absence_list.inc

```diff
@@ -441,32 +292,18 @@ $handler->display->display_options['filters']['absence_start_date_period_filter'
 $handler->display->display_options['filters']['absence_start_date_period_filter']['group_info']['identifier'] = 'absence_start_date_period_filter';
 $handler->display->display_options['filters']['absence_start_date_period_filter']['group_info']['optional'] = FALSE;
 $handler->display->display_options['filters']['absence_start_date_period_filter']['group_info']['default_group'] = '1';
-
-/**
- * Dynamic field creation start - CiviHR
-**/
-
-$current_year = date("Y");
-
 $handler->display->display_options['filters']['absence_start_date_period_filter']['group_info']['group_items'] = array(
   1 => array(
     'title' => 'Generated from code (Do not overwrite)',
     'operator' => 'between',
     'value' => array(
       'type' => 'date',
       'value' => '',
-      'min' => $current_year . '-01-01 00:00:00',
-      'max' => $current_year . '-12-31 23:59:59',
+      'min' => '2016-01-01 00:00:00',
```

Comment:
I know this an auto generated .. but why min and max values are hardcoded here ?